packages/@chakra-icons/cli/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@chakra-icons/cli",
```

Comment:
Are you sure for make scoped for `create-chakra-icons`, since this named has been published and user know the name ?